dnf-behave-tests/createrepo_c/steps/repodata.py

```diff
@@ -176,19 +176,19 @@ def repodata_in_path_is(context, path):
                                  " not match suffix of File: " + repodata_file)
         try:
             tmp = next(decompression_iter(filepath, compression_type, blocksize=100))
+            if filepath.endswith(compression_suffix):
+                    filepath = filepath[:-(len(compression_suffix))]
             if tmp:
-                if filepath.endswith(".zck"):
-                    assert("ZCK" in str(tmp))
-                elif filepath.endswith(".sqlite"):
-                    assert("SQLITE" in str(tmp))
+                if filepath.endswith(".sqlite"):
+                    assert("SQLite" in str(tmp))
```

Comment:
You've changed the case on the "SQLITE" string, is it intentional? ("SQLITE" before, "SQLite" after)

Reply:
Yea, there was a bug actually. it always ended in the `else` branch and just passed. :-( Even if the string was wrongly capitalized. I will extract it into a separate commit and fix the dangerous default `pass`.